package.json

```diff
@@ -37,39 +36,46 @@
     "@storybook/addon-centered": "^v3.1.2",
     "@storybook/react": "^v3.1.7",
     "babel-core": "^6.4.5",
+    "babel-eslint": "^7.2.3",
     "babel-loader": "^7.1.1",
     "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-react-remove-prop-types": "^3.0.0",
     "babel-plugin-react-require": "^3.0.0",
     "babel-plugin-react-transform": "^2.0.2",
     "babel-plugin-transform-dead-code-elimination": "^2.0.2",
-    "babel-plugin-transform-inline-environment-variables": "^6.8.0",
+    "babel-plugin-transform-inline-environment-variables": "^0.1.1",
     "babel-plugin-transform-react-constant-elements": "^6.9.1",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.6",
     "babel-preset-es2015-loose": "^8.0.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.5.0",
     "babel-traverse": "^6.14.0",
     "coveralls": "^2.11.12",
-    "enzyme": "^2.4.1",
+    "enzyme": "^2.9.1",
+    "eslint": "^3.19.0",
```

Comment:
copied them from there, but there was hundreds of failing lints. I'll do the TS=>JS first then run prettier over it in another MR